package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "@babel/preset-typescript": "^7.22.5",
     "@playwright/test": "^1.37.0",
+    "@replayio/playwright": "3.0.0-alpha.6",
```

Comment:
this could now be bumped to `.7`

Reply:
Gotcha. Just bumped the version.

Noticed that I can't run e2e tests locally though:
```sh
$ p test:e2e        

> react-resizable-panels-website@0.0.1 test:e2e /Users/bvaughn/Documents/git/oss/react-resizable-panels/packages/react-resizable-panels-website
> playwright test

[replay.io]: Failed to send test metrics
[replay.io]: Error: request to https://webhooks.replay.io/api/metrics failed, reason: socket hang up
[replay.io]: 🕑 Completing some outstanding work ...
[replay.io]: 
[replay.io]: ❌ We encountered some unexpected errors processing your recordings and was unable to upload them.
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
[replay.io]:    - Failed to upload recording: Cannot connect to server wss://dispatch.replay.io
```

Maybe this is because I didn't pass the api token env var? But the error message isn't great if that's the case, and this would mean that no contributors could run these tests either.

Reply:
Also the terminal was totally silent for several seconds before printing this. I'm not sure if that meant tests weren't running, or if it was just swallowing the `stdout` output. (That output is important to debug.)

Reply:
So you have this error, CI reports "RecordingUploadError: Failed to set recording metadata Error: Protocol client is not initialized" and I have "RecordingUploadError: Failed to set recording metadata Protocol error 1: Internal error" locally now 😂 

Well, I'm investigating all of this now ;p